ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -123,6 +123,10 @@ def get_external_cluster_details(self):
                 f"--rgw-zone-name {rgw_zone}"
             )
 
+        if config.EXTERNAL_MODE.get("run_as_user"):
```

Comment:
Please check also upgrade path as IIRC the run_as_user was not passed to upgrade script since 4.11